--- c94008c.adb.orig	2017-11-05 17:08:10.000000000 +0900
+++ c94008c.adb	2017-11-05 17:08:53.000000000 +0900
@@ -198,10 +198,10 @@
                     OR WHEN ENTER_TERMINATE => TERMINATE;
                     END SELECT;
 
-                    DELAY 10.0;
+                    DELAY 1.0;
 
                     IF TERMINATE_COUNT.GET /= 1 THEN
-                         DELAY 20.0;
+                         DELAY 2.0;
                     END IF;
 
                     IF TERMINATE_COUNT.GET /= 1 THEN
@@ -239,10 +239,10 @@
 
      BEGIN
 
-          DELAY 10.0; -- WAIT FOR T1, T2, AND T3 TO GET TO SELECT STMTS.
+          DELAY 1.0; -- WAIT FOR T1, T2, AND T3 TO GET TO SELECT STMTS.
 
            IF TERMINATE_COUNT.GET /= 3 THEN
-                DELAY 20.0;
+                DELAY 2.0;
            END IF;
 
            IF TERMINATE_COUNT.GET /= 3 THEN
