--- c94001b.adb.orig	2014-09-02 23:37:57.000000000 +0900
+++ c94001b.adb	2014-09-02 23:38:21.000000000 +0900
@@ -63,7 +63,7 @@
                ACCEPT E (I : INTEGER) DO
                     LOCAL := I;
                END E;
-               DELAY 30.0;    -- SINCE THE PARENT UNIT HAS HIGHER
+               DELAY 3.0;     -- SINCE THE PARENT UNIT HAS HIGHER
                               -- PRIORITY AT THIS POINT, IT WILL
                               -- RECEIVE CONTROL AND TERMINATE IF
                               -- THE ERROR IS PRESENT.
