--- support/impdef-annex_c.adb.orig	2013-04-06 09:14:25.000000000 +0900
+++ support/impdef-annex_c.adb	2013-04-06 09:15:31.000000000 +0900
@@ -42,8 +42,10 @@
    -- possibility.
 
    procedure Generate_Interrupt is
+      procedure C_raise (sig : Ada.Interrupts.Interrupt_Id);
+      pragma Import (C, C_raise, "raise");
    begin
-      Report.Comment (". >>>>> GENERATE THE INTERRUPT NOW <<<<< ");
+      C_raise (Interrupt_To_Generate);
 
    -- ^^^^^^^^^^^^^^^^^^^^  MODIFY THIS BODY AS NEEDED  ^^^^^^^^^^^^^^^^^^^^
 
