--- support.build/impdef-annex_c.ads.orig	2013-03-07 02:58:47.000000000 +0900
+++ support.build/impdef-annex_c.ads	2013-03-07 02:59:52.000000000 +0900
@@ -52,7 +52,7 @@
    -- (as specified by the constant Wait_For_Interrupt) during testing.
 
    Interrupt_To_Generate: constant Ada.Interrupts.Interrupt_ID :=
-      Ada.Interrupts.Interrupt_ID'First;  -- to allow trivial compilation
+      Ada.Interrupts.Names.SIGUSR1;  -- to allow trivial compilation
    -- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^  --- MODIFY HERE AS NEEDED
    
 --=====-=====-=====-=====-=====-=====-=====-=====-=====-=====-=====-=====--
