--- support.build/report.adb.orig	2015-03-30 09:35:08.000000000 +0900
+++ support.build/report.adb	2015-03-30 09:36:33.000000000 +0900
@@ -62,6 +62,7 @@
 --      RLB  3/22/07  UPDATED ACATS VERSION STRING TO "3.0".
 --      RLB  1/20/14  UPDATED ACATS VERSION STRING TO "3.1".
 
+with Ada.Command_Line;
 WITH TEXT_IO, CALENDAR;
 USE TEXT_IO, CALENDAR;
 PRAGMA ELABORATE (TEXT_IO, CALENDAR);
@@ -217,14 +218,17 @@
           WHEN DOES_NOT_APPLY =>
                PUT_MSG ("++++ " & TEST_NAME (1..TEST_NAME_LEN) &
                         " NOT-APPLICABLE ++++++++++++++++++++.");
+               Ada.Command_Line.Set_Exit_Status (2);
           WHEN ACTION_REQUIRED =>
                PUT_MSG ("!!!! " & TEST_NAME (1..TEST_NAME_LEN) &
                         " TENTATIVELY PASSED !!!!!!!!!!!!!!!!.");
                PUT_MSG ("!!!! " & (1..TEST_NAME_LEN => ' ') &
                         " SEE '!' COMMENTS FOR SPECIAL NOTES!!");
+               Ada.Command_Line.Set_Exit_Status (3);
           WHEN OTHERS =>
                PUT_MSG ("**** " & TEST_NAME (1..TEST_NAME_LEN) &
                         " FAILED ****************************.");
+               Ada.Command_Line.Set_Exit_Status (4);
           END CASE;
           TEST_STATUS := FAIL;
           TEST_NAME_LEN := NO_NAME'LENGTH;
@@ -283,6 +287,7 @@
      END IDENT_WIDE_STR;
 
      FUNCTION EQUAL (X, Y : INTEGER) RETURN BOOLEAN IS
+          pragma Unsuppress (Overflow_Check);
           REC_LIMIT : CONSTANT INTEGER RANGE 1..100 := 3;  -- RECURSION
                                         -- LIMIT.
           Z : BOOLEAN;                  -- RESULT.
